lib/mauth/client.rb

```diff
@@ -299,13 +304,17 @@ def authenticate!(object)
         time_valid!(object)
         token_valid!(object)
         signature_valid!(object)
-      rescue InauthenticError
+      rescue MauthNotPresent => e
+        logger.warn "mAuth signature authentication failed for #{object.class}."
+        logger.warn e.message
```

Comment:
What do you think about combining these two log lines (here and also for the exceptions below)?  Each line we log has significant overhead so it seems more efficient to combine them and also if they are from the same event, I think it is nice to have them together in the same line in the alert email (should someone choose to alert on these events).

Reply:
I'm in favor of combining the lines. The main concern as I see it would be breaking log alerts but I think after the next mauth release we will have to take another look at mauth alerts anyway and write some new ones for MWSV2.

Reply:
Done!